keps/prod-readiness/sig-scheduling/3633.yaml

```diff
@@ -0,0 +1,3 @@
+kep-number: 3633
+alpha:
+  approver: TBD
```

Comment:
when sig-scheduling finishes review, we should seek a PRR reviewer. maybe @wojtek-t

Reply:
Hello @wojtek-t, I assigned this KEP to you. Please teach us if another person is suitable here. Thanks 🙏